src/database/db_build.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+const db_connection = require("./db_connection")
+const build_file = fs.readFileSync(__dirname + "/db_build.sql", "utf8");
+
+db_connection.query(build_file, (error, result)=>{
+  if (error) {
+    console.log("Failed", error);
+  }else {
+    console.log("Success!");
+  }
+});
```

Comment:
You should check for the node enviroment, and to export it as moudle when the enviroment is testing while to run it otherwie.
Consider doing that for when writing the tests